src/index.ts

```diff
@@ -229,8 +257,19 @@ export function createBirpc<RemoteFunctions = Record<string, never>, LocalFuncti
       else {
         try {
           result = await fn.apply(rpc, args)
+
+          // handle async iter result
+          if (result?.[Symbol.asyncIterator]) {
+            for await (const iterRes of result) {
+              if (msg.i)
+                post(serialize(<Response>{ t: 's', i: msg.i, r: iterRes, d: false, e: error }), ...extra)
+            }
+            if (msg.i)
+              return post(serialize(<Response>{ t: 's', i: msg.i, r: undefined, d: true, e: error }), ...extra)
```

Comment:
I'd say we probably only return the `d` flat when it's not finished to save some bytes for regular calls. Maybe `u` for `unfished` or `s` for `stream`?

Reply:
Currently, `d` flag would be only posted when fn return result is iterable. In case of regular calls, it would be `undefined`. Is this u expect?